src/pages/kyunghyun/Login/Login.js

```diff
@@ -1,88 +1,102 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+// import { Link } from 'react-router-dom';
 import './Login.scss';
 import '../../../styles/common.scss';
-import LoginForm from './LoginForm';
 class Login extends React.Component {
   constructor() {
     super();
     this.state = {
-      idValue: '',
-      pwValue: '',
-      isValid: false,
+      id: '',
+      pw: '',
+      valid: false,
     };
   }
 
-  handleIdInput = e => {
+  handleInput = e => {
+    const { value, name } = e.target;
     this.setState({
-      idValue: e.target.value,
+      [name]: value,
     });
-  };
-
-  handlePwInput = e => {
-    this.setState({
-      pwValue: e.target.value,
-    });
-  };
+  }; // id/pw 통합 input핸들러 : state객체 속성을 이용 []:value 방식으로 적용되는 input을 나눔
 
   checkValid = e => {
-    const { idValue, pwValue } = this.state;
-    const validId = idValue.includes('@');
-    const validPw = pwValue.length >= 6;
+    const { id, pw } = this.state;
+    const validId = id.includes('@');
+    const validPw = pw.length >= 6;
```

Comment:
변수를 하나로 합치거나 안 적으시는 것도 나쁘지 않을 것 같습니다

Reply:
```suggestion
    const isValidId = id.includes('@');
    const isValidPw = pw.length >= 6;
```
저는 변수 분리하는 거 괜찮아보이긴 하는데 is 네이밍이 붙으면 좋을거 같아요